circle.yml

```diff
@@ -13,6 +13,7 @@ test:
   override:
     - >
       ./etc/scripts/docker/run.sh
+      --debug
```

Comment:
Hey @timraymond, yup, I added those because @rossmcdonald 's build scripts report more information about what commands are running.  

I'm a bit old school this way, but, I like very chatty builds especially on CI systems.